app/controllers/authenticated_controller.rb

```diff
@@ -19,12 +20,6 @@ class AuthenticatedController < ApplicationController
     raise ActiveRecord::RecordNotFound.new
   end
 
```

Comment:
The reason I moved this into a helper was so that I can call it from within `link_to_conflicting_revision`[https://github.com/dradis/dradis-ce/pull/22/files#diff-e86fe144a2807be4de67fe4095611223R37], which needs to be available in the views. 

I can't see a way of making that method aware of whether or not a revision is from the current user or not without duplicating the logic in `user_for_paper_trail`... and if we're going to duplicate the logic, we might as well just call the method.

But I suppose I don't have to *move* the method like this; I could keep in the controller but make it available to the views using `helper_method`, [like this](http://stackoverflow.com/a/8908087/1603071).

What do you think?